example/notify_inval_entry.c

```diff
@@ -78,13 +78,16 @@
 
 #define FUSE_USE_VERSION 34
 
+#include <fuse_i.h>
```

Comment:
Why is this needed? That should only be used internally in libfuse, not in an example.

Reply:
needed it for `se->error = ret;`
 https://github.com/libfuse/libfuse/blob/0cc6fe0d9b560786e63a075c77d5014e7bca1590/example/notify_inval_entry.c#L270-L276
 
i will remove it as with the current design it is not needed anymore.